tests/unit/test_spl_token_instructions.py

```diff
@@ -57,6 +57,16 @@ def test_initialize_multisig():
     assert spl_token.decode_initialize_multisig(instruction) == params
 
 
+def test_create_associated_token_account(stubbed_sender):
+    """Test create associated token account."""
+    mint = PublicKey(0)
```

Comment:
Integration test failed. Looks like using `PublicKey(0)` as the `mint` address won't work. You need an actual spl token mint address.

Reply:
You should call this method to create an SPL token mint:
https://michaelhly.github.io/solana-py/spl.html#spl.token.client.Token.create_mint

Reply:
So I was actually able to use the stubbed_sender as the payer and mint, and make the owner the PublicKey(0) which it was happier with.

Reply:
Right. But I don't think this is proper, can you use `Test_token.pubkey`?

Reply:
Ahh sorry. This wasn't the test that was broken. My bad.